src/internal/transactionenv/txncontext/context.go

```diff
@@ -64,29 +58,13 @@ func (t *TransactionContext) Finish() error {
 	return nil
 }
 
-// FinishPipelineCommits saves a pipeline output branch to have its commits
-// finished at the end of the transaction
-func (t *TransactionContext) FinishPipelineCommits(branch *pfs.Branch) error {
-	if t.CommitFinisher != nil {
-		return t.CommitFinisher.FinishPipelineCommits(branch)
-	}
-	return nil
-}
-
 // PfsPropagater is the interface that PFS implements to propagate commits at
 // the end of a transaction.  It is defined here to avoid a circular dependency.
 type PfsPropagater interface {
 	PropagateBranch(branch *pfs.Branch) error
 	Run() error
 }
 
-// PipelineCommitFinisher is an interface to facilitate finishing pipeline commits
-// at the end of a transaction
-type PipelineCommitFinisher interface {
```

Comment:
The `CommitFinisher` was kind of extraneous now - added handling in PPS that stops associated jobs when updating a pipeline.